maps/southern_cross/items/clothing/sc_under.dm

```diff
@@ -7,6 +7,9 @@
 	worn_state = "pilot1"
 	item_icons = list(slot_w_uniform_str = 'maps/southern_cross/icons/mob/sc_under.dmi')
 	icon = 'maps/southern_cross/icons/obj/sc_uniforms.dmi'
+	sprite_sheets = list(
```

Comment:
I think this is inherited from the default under clothing item?

Reply:
This is specifically for the southern-cross-uniform DMI